### ISSUE TITLE
Pywal doesn't wok with kitty terminal

### ISSUE THREAD

**Issue by xa0s-0** [dylanaraps/pywal#215]
Hi!

I set up everything. Here is what I put into `/etc/profile` and `~/.profile`: 
`(cat ~/.cache/wal/sequences &)`
`cat ~/.cache/wal/sequences`
`source ~/.cache/wal/colors-tty.sh`

Then I added `wal -R` into `~/.xinitrc` and pywal is not working. When I type command `wal -i mywal.png` everything works fine, but when I open another terminal it's using default colors. Did I do something wrong?

Here is my spec:
os: Arch Linux 
wm: i3-gaps 
terminal: kitty-terminal

**Comment by dylanaraps** [comment]
Kitty most likely doesn't support the escape sequences required. This needs to be fixed on their end.

You can open an issue on their repo, see this issue as an example: https://github.com/jwilm/alacritty/issues/656

**Comment by xa0s-0** [comment]
Yeah, I think that the problem is with fish. When I changed terminal to `zsh` everything works. Do you know how can I set up pywal on fish?

**Comment by dylanaraps** [comment]
I’ll take a look and see if I can reproduce this.

**Comment by piggyslasher** [comment]
Weird. For me, it's working great on Arch, using cat...sequences but one OSX, it's not doing anything. iTerm works though. And alacritty.

I'm using the same config between Arch & Osx.

**Comment by dylanaraps** [comment]
It works fine for me even with `fish`. I don't think this is a `pywal` issue.

**Comment by paanvaannd** [comment]
@piggyslasher Did you get it to work on macOS? It works flawlessly with iTerm 2 but Kitty's theme does not change from the one that I have specified in the kitty.conf file. Are there extra steps one must take on macOS in order to get the theme to match the pywal-generated colorscheme?

**Comment by spolakh** [comment]
I managed to ~ make it work on macOS. First, I had to install from git since the pip version was way out of date and had a couple of macos-specific troubles. After that running `wal -R` (assuming you already have generated theme already) /from kitty itself/ works. I'm still trying to figure out why doesn't it work if executed from outside of kitty since my driver script that manages pywal is running in a daemon. Would be happy to get any pointers.

**Comment by spolakh** [comment]
I've figured out why `wal -R` only worked form within kitty: because by default kitty remote control expects to be ran from within one of the kitty windows. Added #595 to communicate over the socket if calling from outside

**Comment by Kevin-Mok** [comment]
Adding `wal -r` to my `~/.config/kitty/kitty.conf` worked for me.
